README.md

```diff
@@ -125,3 +138,13 @@ or
 ```sh
 make test.e2e.ocp
 ```
+
+## Community Support and Contributing
+Please refer to the [CONTRIBUTING-SAIL-PROJECT.md](https://github.com/istio-ecosystem/sail-operator/blob/main/CONTRIBUTING.md) file for more information on how to contribute to the Sail Operator project. This file contains all the information you need to get started with contributing to the project.
+
+If you want to make contributions to the Istio project please refer to the [CONTRIBUTING.md](https://github.com/istio-ecosystem/sail-operator/blob/main/CONTRIBUTING.md) for more information.
```

Comment:
Let's not link to that file, I honestly think it doesn't belong in our repo -and I know Jonh disagrees, but I think people should just read the version that lives in the actual istio repo if they want to contribute to istio

Reply:
Done